changelog.d/4779.misc

```diff
@@ -0,0 +1 @@
+Swap out riot.im icons and logos for a more static resource.
```

Comment:
the icons and logos are staying the same - it's just the urls that are changing, no? Or am I misunderstanding?

Reply:
Some of them have changed to the new logo and colour scheme, but most are just moved.

Reply:
I'm also concerned that the mention of riot.im in our changelog will set alarm bells ringing, not without reason.

```suggestion
Update URLs for riot.im icons and logos in the default notification templates.
```